caffe2/operators/batch_box_cox_op.cc

```diff
@@ -301,7 +301,7 @@ void BatchBoxCoxOp<CPUContext>::BoxCoxMixedLambda(
 #define DEFINE_CACHED_BUFFERS(T, tag)                                         \
   template <>                                                                 \
   template <>                                                                 \
-  BatchBoxCoxOp<CPUContext>::TypedCachedBuffers<T>&                           \
+  CAFFE2_API BatchBoxCoxOp<CPUContext>::TypedCachedBuffers<T>&                           \
```

Comment:
CAFFE2_API can actually be referring to C10_IMPORT or C10_EXPORT [based on the backend used](https://github.com/pytorch/pytorch/blob/master/c10/macros/Export.h#L96).

I was wondering if it would be better to just directly use `C10_EXPORT`

Reply:
Thanks reminder. updated